### PR TITLE
[#611] Froze python packages versions in Airflow dockerfile

### LINKED ISSUE #611: Sample Airflow DAGs hang without any errors
I noticed that starting from 16Nov build, Airflow DAGs tasks hang in **in progress** status and never ends. Airflow Worker logs are empty. There are no errors in logs of any Airflow pods.
After long investigation I found that new version of **redis-py** python package released and it's updated from 2.10.6 to 3.0.1. https://github.com/andymccurdy/redis-py
It could be the reason of that issue.
IMHO we need to freeze **redis-py** package and extend tests to check if sample dags are completed in several minutes.

### PR DESCRIPTION
**Pull request by aliaksandr-d**
This PR closes #611 
Froze python packages versions in Airflow dockerfile and changed tests to check if any dags succeeded

### CODE REVIEW

**Comment by aliaksandr-d** [comment]
bot, test apps

File: k8s/airflow/Dockerfile [hunk 1]
```diff
@@ -29,9 +29,9 @@ RUN apt-get update && apt-get install -y \
 RUN service cron stop \
     && useradd -ms /bin/bash -d ${AIRFLOW_HOME} airflow
 
-RUN pip3 install --disable-pip-version-check cryptography \
+RUN pip3 install --disable-pip-version-check cryptography==2.4.1 \
     && pip3 install --disable-pip-version-check apache-airflow[postgres,jdbc,password,mysql,celery,redis,s3]==1.9.0 \
-    pyOpenSSL ndg-httpsclient pyasn1 psycopg2-binary redis hiredis requests
+    pyOpenSSL==18.0.0 ndg-httpsclient==0.5.1 pyasn1 psycopg2-binary==2.7.6.1 redis==2.10.6 hiredis==0.2.0 requests
```
Comment (dsuslov) [inline k8s/airflow/Dockerfile]: @aliaksandr-d please pin version of all dependencies. pyasn1 == ?

**Comment by aliaksandr-d** [comment]
bot, test apps

**Comment by aliaksandr-d** [comment]
bot, test all

File: k8s/airflow/Dockerfile (hunk 1 above)
Comment (aliaksandr-d) [reply k8s/airflow/Dockerfile]: Fixed

Comment (aliaksandr-d) [reply k8s/airflow/Dockerfile]: Fixed

**Pull request merged**